build.py

```diff
@@ -309,6 +316,9 @@ def clone_repo(full_repo, target_dir=None, version=None, branch=None):
         execute(['git', 'reset', '--hard', version])
```

Comment:
Addressed in 	`4ed25db`.

When testing this PR, specify the `caseypt` user in the command. I updated the config there to exercise the `version` argument.